### ISSUE TITLE
Add Yii2 API controller

### ISSUE THREAD

**Issue by TheAdnan** [TheAdnan/Yii1-API-controller#1]
- Add a new folder named `yii2`

- Inside it add the ApiController for Yii2 framework (a couple of tweaks here and there, but generally an easy task)

- Add a README (use the initial README as a template)

**Comment by xepozz** [comment]
I'll take this task

**Comment by xepozz** [comment]
Hm.. do I need to move ApiController and README to folder yii2?

**Comment by TheAdnan** [comment]
Yeah, but keep the README and ApiController for Yii1 in the root folder.

**Comment by archlemon** [comment]
What would the controller need to do? All the methods described in your Yii1 controller are already covered in Yii2 via `yii\rest\ActiveController` and `yii\rest\Controller`
